### DISCUSSION TITLE
SurfaceTension_CoveredSphere_ZR2020

### DISCUSSION THREAD

**Discussion by hosseinrezaa** [InteractiveComputerGraphics/SPlisHSPlasH#173] [Q&A]
I checked the input file for surface tension. Is there any document describes these parameters used in the code?

	"Materials": [
		{
			"id": "Fluid",
			"surfaceTension": 0.1,
			"surfaceTensionMethod": 4,
			"viscosity": 0.01,
			"viscosityMethod": 1,
			"surfTZRversion": 0,
			"surfTZRCsd": 10000,
			"surfTZRr-ratio": 0.8,
			"surfTZRtau": 0.5,
			"surfTZRd": 28,
			"surfTZRsampling": 0 ,
			"surfTZRnormal-mode": 1,
			"surfTZRd-offset": 3,
			"surfTZRPcaMixNrm": 0.8,
			"surfTZRPcaMixCur": 0.4,
			"surfTZRMCSamples": -1 ,
			"surfTZRNeighLimit": 16	
		}

Moreover, is it possible to consider different surface tensions for the solid bodies (for example different surface tensions for the solid sphere)? or it is excluded for the fluids?

**Comment by janbender** [comment]
All parameters are described in the documentation:
https://splishsplash.readthedocs.io

What do you mean with solid? A rigid body does not have a surface tension since it is rigid.

**Comment by hosseinrezaa** [comment]
Solid surfaces with different materials show different behaviors. Some surfaces are hydrophobic and other ones are hydrophilic, how should I apply these behaviors?

**Comment by janbender** [comment]
There is only some adhesion force in some surface tension models. However, if you need something else, the best way is to implement an own nonpressure force:

https://splishsplash.readthedocs.io/en/latest/implement_nonpressure_force.html
